hibernate-core/src/main/java/org/hibernate/dialect/SybaseDialect.java

```diff
@@ -60,13 +60,15 @@
  */
 public class SybaseDialect extends AbstractTransactSQLDialect {
 
-	protected boolean jtdsDriver;
+	protected boolean jtdsDriver = true;
```

Comment:
Why not just set the field to true, instead of doing it in every constructor?
Reminder: we set it to 'true' to solve a problem (don't recall what that was)

Reply:
Because we want this information to be immutable. Setting this to true is necessary when we have no JDBC metadata available, because the JTDS driver is limited, and we have to be pessimistic about driver support when we have no JDBC metadata.

Reply:
Making the field final is fine, that was not what my question was about; i was rather referring to the fact that we set it by default to true (as per your suggestion) to fix some failing test(s) - again, don't remember the actual cause. IIUC you want me to undo this default true and revert back to what there was before,as you indicate in https://github.com/hibernate/hibernate-orm/pull/5141#discussion_r962656375 ?

Reply:
In the constructor that receives the `DialectResolutionInfo`, the field should be initialized to `jtdsDriver = info.getDriverName() != null && info.getDriverName().contains( "jTDS" );` and in all the other constructors, it should be initialized to `true`.